### DISCUSSION TITLE
Hidden text in textarea component

### DISCUSSION THREAD

**Discussion by mschasz** [twilio-labs/paste#3415] [General] [answered]
Hi, 

I'm building out a workflow to create an OAuth application in Segment. As part of this workflow, we allow users to revoke a token, which is entered in a `textarea` component. We'd like to make this input hidden, but it doesn't look like that's an option for this component. Is the guidance to use the `input` component, which allows users to hide their input, instead? Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hi @mschasz 

I'm assuming the user isn't actually entering data into the hidden field as that would be a pretty tricky thing to do if you can't see the input, but instead its used by you as part of the flow?

If you don't see the input, I'm not entirely sure of the restriction on needing a textarea, wouldn't a hidden input function exactly the same for that purpose? Just somewhere you can place a value and retrieve it later?

Going further down that line of thought, if it's hidden, a `<Box as="textarea" hidden />` would probably work just fine too, as would just plain html elements `<input type="hidden" />`. At that stage theres no real requirement it needs to be a component, an HTML element will suffice. 

To quote a little MDN from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden

> `<input>` elements of type hidden let web developers include data that cannot be seen or modified by users when a form is submitted. For example, the ID of the content that is currently being ordered or edited, or a unique security token

That might help point you in a direction too.
